src/Abp.RedisCache/AbpRedisCacheConfig.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Abp.Configuration.Startup;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Abp
+{
+    public class AbpRedisCacheConfig
+    {
+        private string connectionStringKey = "Abp.Redis.Cache";
+
+        public string ConnectionStringKey
+        {
+            get { return connectionStringKey; }
+            set { connectionStringKey = value; }
+        }
+
+
+    } 
+
+    public static class AbpRedisCacheConfigEctensions
```

Comment:
is it possible to fix spelling or Ectensions should be Extensions (with an x) so it becomes AbpRedisCacheConfigExtensions